ec2/InstanceType.dhall

```diff
@@ -389,70 +410,268 @@ let alternatives =
+                                , InstanceType.c5n-4xlarge
+                                ]
+                              # ( if    _params.permitLowerEBSBandwidth
+                                  then  lowDiskBandwidth
+                                  else  [] : List InstanceType
+                                )
 
                     in    instancefamily
                         # ( if    _params.permitCrossInstanceFamily
-                            then    cross-instances
-                                  # ( if        _params.permitLowerClassCPU
-                                            &&  _params.permitWorseNetwork
-                                      then  [ InstanceType.m4-4xlarge ]
-                                      else  [] : List InstanceType
-                                    )
+                            then  cross-instances
+                            else  [] : List InstanceType
+                          )
+                , c5a-large =
+                    let cross-instances =
+                          let lowClass =
+
+                              {-
+                                lowClass - all instance types that have lower ECU and lower network performance
+                                  -}
+                                [ InstanceType.m4-large ]
+
+                          in    [ InstanceType.m5a-large
+                                , InstanceType.r5a-large
+                                ]
+                              # ( if        _params.permitLowerClassCPU
+                                        &&  _params.permitWorseNetwork
+                                  then  lowClass
+                                  else  [] : List InstanceType
+                                )
+
+                    let instancefamily =
+                          [ InstanceType.c5-large
+                          , InstanceType.c5d-large
+                          , InstanceType.c5n-large
+                          ]
+
+                    in    instancefamily
+                        # ( if    _params.permitCrossInstanceFamily
+                            then  cross-instances
+                            else  [] : List InstanceType
+                          )
+                , c5a-xlarge =
+                    let cross-instances =
+                          let lowClass =
+
+                              {-
+                                lowClass - all instance types that have lower ECU and lower network performance
+                                  -}
+                                [ InstanceType.m4-xlarge ]
+
+                          in    [ InstanceType.m5a-xlarge
+                                , InstanceType.r5a-xlarge
+                                ]
+                              # ( if        _params.permitLowerClassCPU
+                                        &&  _params.permitWorseNetwork
+                                  then  lowClass
```

Comment:
You need to have permitLowerEBSBandwidth here too because c5a.xlarge has EBS bandwidth up to 3170 Mbps while m4.large has EBS bandwidth of 750 Mbps dedicated